custom/assets/js/calculateBodySurfaceAreaMosteller.js

```diff
@@ -0,0 +1,101 @@
+function calculateBodySurfaceAreaMosteller(height, height2, heightUnit, weight, weightUnit, returnUnit) {
+/*
+@package       OpenEMR
+@author        Rachel Ellison <ellison.rachel.e@gmail.com>
+@copyright (C) 2020 Rachel Ellison <ellison.rachel.e@gmail.com>
+@link          http://www.open-emr.org
+@license       https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+
+
+This function is used to calculate an approximate body surface area for a person given their height and weight by implementing the Mosteller formula.
+It is the square root of the height(in centimeters) multiplied by the weight(in kilograms) divided by 3600.
+
+This function supports heights in units of centimeters, meters, inches, and feet + inches.
+This function supports weights in units of pounds, kilograms, and stone.
+
+You need to specify the units and provide them according to the Unified Code for Units of Measure (UCUM) standard.
+https://ucum.nlm.nih.gov/
+https://loinc.org/usage/units/
+
+This link is pretty dated but I believe the codes are still accurate.
+https://ucum.nlm.nih.gov/example-UCUM-Codes-v1.4.pdf
+
+
+height 2 should not be used unless you're entering in feet and inches separately.
+I always set it to 0 so I don't forget about it shift everything to the left accidentally.
+*/
+
+//determine unit of measure provided for height and convert to centimeters.
+	switch(heightUnit){
+    	case "cm":
+        	//This is the unit we want, do nothing.
+        break;
+        case "[in_i]":
+        	//in is not valid, you need the underscore i to indicate that it's international inches.

```

Comment:
are the brackets around `in_i` necessary?

Reply:
Yes, I believe it's not a valid UCUM code without them.